oak/server/rust/oak_runtime/src/node/logger.rs

```diff
@@ -36,12 +38,24 @@ fn logger(pretty_name: &str, runtime: RuntimeRef, reader: ChannelReader) -> Resu
         let _ = runtime.wait_on_channels(&[Some(&reader)]);
 
         if let Some(message) = runtime.channel_read(&reader)? {
-            let message = String::from_utf8_lossy(&message.data);
-            info!("{} LOG: {}", pretty_name, message);
+            if let Ok(msg) = LogMessage::decode(&*message.data) {
+                info!(
+                    "{} LOG: {} {}:{}: {}",
+                    pretty_name,
+                    to_level_name(msg.level),
+                    msg.file,
+                    msg.line,
+                    msg.message
+                )
+            };
         }
     }
 }
 
+fn to_level_name(level: i32) -> String {
+    format!("{:?}", Level::from_i32(level).expect("Unknown log level.")).to_ascii_uppercase()
```

Comment:
Is there a better way to do this?

Reply:
`expect` is probably a bit strict &ndash; it means a Node can terminate the runtime by emitting a malformed log message.   Maybe `Level::from_i32(level).unwrap_or(Level::UNKNOWN_LEVEL)` ?

Reply:
Also, do you need `Level::from_i32` and an `i32` typed parameter?  Does `LogMessage::level` start as type `oak::logger::proto::Level` already?

Reply:
I think I need them. `LogMessage::level` in the code generated by `Prost` is `i32`.